src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -243,13 +245,12 @@ uct_rc_mlx5_iface_handle_failure(uct_ib_iface_t *ib_iface, void *arg,
     if (ep == NULL) {
         ucs_diag("ignoring failure on removed qpn 0x%x wqe[%d]", qp_num, pi);
         uct_rc_iface_add_cq_credits_dispatch(iface, 1);
+        ucs_arbiter_dispatch(&iface->tx.arbiter, 1, uct_rc_ep_process_pending,
```

Comment:
why needed here?

Reply:
we removed it from `uct_rc_iface_add_cq_credits_dispatch`

Reply:
good catch, fixed